src/test/java/com/salesforce/cdp/queryservice/core/QueryServiceConnectionTest.java

```diff
@@ -252,25 +244,17 @@ public void testGrpcIsValid() throws SQLException {
         assertThat(connection.isValid(10)).isFalse();
     }
 
-    @Test
```

Comment:
Please add the testcase

Reply:
this test case was for exception case when hyper threw error and then we called trino endpoint in catch block.
that is not valid now, so I removed the test case.